packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -361,10 +366,26 @@ class MessageList extends Widget<Properties> {
       }
     });
 
+    this._setLastMessageGroupClasses();
     // @ts-expect-error
     this._updateLoadingState(isLoading);
   }
 
+  _setLastMessageGroupClasses(): void {
+    this._$content
+      .find(`.${CHAT_LAST_MESSAGEGROUP_ALIGNMENT_START_CLASS}`)
+      .removeClass(CHAT_LAST_MESSAGEGROUP_ALIGNMENT_START_CLASS);
+    this._$content
+      .find(`.${CHAT_LAST_MESSAGEGROUP_ALIGNMENT_END_CLASS}`)
+      .removeClass(CHAT_LAST_MESSAGEGROUP_ALIGNMENT_END_CLASS);
+
+    const $lastAlignmentStartGroup = this._$content.find(`.${CHAT_MESSAGEGROUP_ALIGNMENT_START_CLASS}`).last();
+    const $lastAlignmentEndGroup = this._$content.find(`.${CHAT_MESSAGEGROUP_ALIGNMENT_END_CLASS}`).last();
+
+    $lastAlignmentStartGroup.addClass(CHAT_LAST_MESSAGEGROUP_ALIGNMENT_START_CLASS);
+    $lastAlignmentEndGroup.addClass(CHAT_LAST_MESSAGEGROUP_ALIGNMENT_END_CLASS);
```

Comment:
well, it can be optimized ofc but I agree that's enough for now

Reply:
good

Reply:
I don't like direct interaction with the markup. I don’t have any objections to leaving it as it is if it currently seems like an acceptable.